doc/plugins/org.polarsys.capella.tipsandtricks.doc/html/Tips and tricks/6_Diagram exploration/6_Diagram exploration.mediawiki

```diff
@@ -115,4 +115,12 @@ The new tool '''Representation Link''' in the palette allows to create a graphic
 ==Refresh many representations at once==
 On the '''Project Explorer''', it is possible to refresh at once many ''sub''-representations related to a selected element.
 
-[[File:../../Images/Tips92.png|thumbnail|480px]]
\ No newline at end of file
+[[File:../../Images/Tips92.png|thumbnail|480px]]
+
+==Title Blocks==
+The title blocks enables the possibility to add information about diagram elements or the diagram itself and can be shown in diagram to provide additional information chosen by users.
+Additional details about title blocks (tools, customization) can be found below:
+* [https://wiki.corp.thales/pages/viewpage.action?pageId=785879073#id-2020What'snewinCapella5.0?-TitleBlocks Capella what's new page]

```

Comment:
We can't use an intranet link on opensource ;)
we can use https://github.com/eclipse/capella/wiki/Release-Notes-1.4.1 instead